peerinst/static/peerinst/css/landing_page.scss

```diff
@@ -23,7 +23,8 @@ header {
   margin-top: 10px;
 }
 .site-content {
-  margin-bottom: 80pt;
+  padding: 0;
+  // margin-bottom: 80pt;
```

Comment:
There was a bottom margin for spacing reasons.  How does this look without it?

Reply:
That margin can be put back. I'm not sure why it was removed.

Reply:
It was because that margin had to be put somewhere else.